mantidproject/mantid
# Refactor SNAPReduce

Like the description says, this takes out the middle of `SNAPReduce` and replaces it with `AlignAndFocusPowder`. This is to reduce duplicated code.

**Report to:** SNAP team

**To test:**

Try it out and see that it works. Less code was deleted than expected, but the duplication of `AlignAndFocusPowder` is now gone. Run the (normally skipped) system test
```bash
./systemtest --archivesearch -R SNAP
```

*There is no associated issue.*



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
